src/shogun/lib/parameter_observers/ParameterObserverLogger.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+* Written (W) 2019 Giovanni De Toni
+*/
+
+#include <shogun/lib/parameter_observers/ParameterObserverLogger.h>
+//#include <shogun/lib/type_case.h>
+
+using namespace shogun;
+
+ParameterObserverLogger::ParameterObserverLogger() {}
+
+ParameterObserverLogger::ParameterObserverLogger(std::vector<std::string> &parameters)
```

Comment:
we won't have these ctors available in the meta examples since it will be instantiated with a factory which calls the default ctor. So instead, you would use the `add` api (see the multiple kernel learning examples for how it works) to add strings to an internal vector of parameter names